iib/web/models.py

```diff
@@ -1064,6 +1104,7 @@ class RequestRegenerateBundle(Request):
     __mapper_args__ = {
         'polymorphic_identity': RequestTypeMapping.__members__['regenerate_bundle'].value
     }
+    build_tags = None
```

Comment:
If it's present in `Request`, you can simply set `nullable=True` in Request and this won't be required.

Reply:
According to docs sqlalchemy.orm.relationship doesn't accept nullable attribute. Or do you mean something else?